pkg/sdk/parameters.go

```diff
@@ -1,6 +1,382 @@
 package sdk
 
-import "fmt"
+import (
+	"context"
+	"fmt"
+	"strconv"
+)
+
+type Parameters interface {
+	SetAccountParameter(ctx context.Context, parameter AccountParameter, value string) error
+	SetSessionParameterForAccount(ctx context.Context, parameter SessionParameter, value string) error
+	SetObjectParameterForAccount(ctx context.Context, parameter ObjectParameter, value string) error
+}
+
+type parameters struct {
+	client *Client
+}
+
+func (parameters *parameters) SetAccountParameter(ctx context.Context, parameter AccountParameter, value string) error {
+	opts := AlterAccountOptions{Set: &AccountSet{Parameters: &AccountLevelParameters{AccountParameters: &AccountParameters{}}}}
```

Comment:
This is hard to read, can you split this over multiple lines?

Reply:
done